src/Components/MainArea/Bookshelf/Bookcase/Bookcase.styled.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
```

Comment:
nit: The risk in those files is that deleting the component itself, won't delete this code. I believe that [tree shaking](https://medium.com/@netxm/what-is-tree-shaking-de7c6be5cadd) will fix the bundle size, but you still have dead code in your repo.

Reply:
TBH I don't understand it. I know what tree shaking is (I know the concept). Is there a better way for keeping styled component? Do you suggest keeping styled-component in the component itself? When I want to delete the component, I would delete the whole folder anyway (or component and files connected to it). Maybe I'm lacking the experience to understand it. Please explain.